src/adapter/src/catalog/config.rs

```diff
@@ -13,6 +13,7 @@ use std::sync::Arc;
 use std::time::Duration;
 
 use bytesize::ByteSize;
+use mz_sql::session::vars::ConnectionCounter;
```

Comment:
Many of the imports are mis sorted, could these get fixed? See https://github.com/MaterializeInc/materialize/pull/19244 for an in-progress PR that should be the new standard.

Reply:
Fixed